tests/mocha/server/CompetitionTest.js

```diff
@@ -0,0 +1,51 @@
+MochaWeb.testOnly(function() {
+  describe('Competition', function() {
+
+    describe('validation', function() {
+      var event, comp;
+
+      it('includes existing events', function() {
+        comp  = make(Competitions, {calendarStartMinutes: 800, calendarEndMinutes: 1100, numberOfDays: 2});
```

Comment:
Spacing nitpick.

Would it make more sense to do `var comp = ...` and `var event = ...` here instead? I'm not sure I see the benefit of line 5.